src/modules/reporting/selectors/select-dispute-markets.js

```diff
@@ -0,0 +1,18 @@
+import { createSelector } from 'reselect'
+import { selectMarkets } from 'modules/markets/selectors/markets-all'
+import { constants } from 'services/augurjs'
+import store from 'src/store'
+
+export default function () {
+  return selectMarketsInDispute(store.getState())
+}
+
+export const selectMarketsInDispute = createSelector(
+  selectMarkets,
+  (markets) => {
+    if (!markets || markets.length === 0) {
```

Comment:
You can use `lodash` `isEmpty` here — I’ve been progressively moving that direction in course to clean up these kinds of conditionals.